Cognite.Simulator.Utils/ProcessUtils.cs

```diff
@@ -0,0 +1,75 @@
+using System.Linq;
+using System;
+using Microsoft.Extensions.Logging;
+using System.Diagnostics;
+using System.Collections.Generic;
+using System.Runtime.Versioning;
+using Cognite.Simulator.Utils.Automation;
+using Cognite.Simulator.Utils;
+using System.Threading;
+using System.Globalization;
+using System.Management;
+using System.Security.Principal;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace Cognite.Simulator.Utils
+{
+     public static class ProcessUtils {
+        public static string GetProcessOwnerWmi(int processId)
+        {
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows)) {
+                throw new PlatformNotSupportedException("This method is only supported on Windows");
+            }
+            string query = $"SELECT * FROM Win32_Process WHERE ProcessId = {processId}";
+            using (var searcher = new ManagementObjectSearcher(query))
+            using (var results = searcher.Get())
+            {
+                foreach (ManagementObject process in results)
+                {
+                    try
+                    {
+                        object[] args = new object[] { "", "" };
+                        if (Convert.ToInt32(process.InvokeMethod("GetOwner", args)) == 0)
+                        {
+                            string user = (string)args[0];
+                            string domain = (string)args[1];
+                            return $"{domain}\\{user}";
+                        }
+                    }
+                    catch
+                    {
+                        return "Access Denied or Process Exited";
```

Comment:
return N/A

Reply:
as we discussed, we can just throw an error and catch it in the caller